src/views/apply/index.html

```diff
@@ -3,8 +3,13 @@
 {% block pageHeading %}Apply to Hack Cambridge{% endblock %}
 {% block pageContent %}
 <div class="content-wrapper-center">
-  <a href="/apply/form" class="button">
-    Log in with MyMLH
+  To apply, or to continue your application, please log in with MyMLH
+  <br>
+  <br>
+  <a href="/apply/form">
+    <button class="button">
+      Log in with MyMLH
+    </button>
```

Comment:
Could we change this back to a `<a class="button">`? Technically this inline-block in inline isn't legal.